### PR TITLE
test: [M3-7957] - Add Cypress integration test for SSH key update and delete

### PR DESCRIPTION
**Pull request by cliu-akamai**
## Description 📝
Add integration test to update and delete SSH key via Profile page.

## Major Changes 🔄
- Check users can update the label of SSH key through Profile page.
- Check users can delete SSH key through Profile page.

## How to test 🧪
```
yarn cy:run -s "cypress/e2e/core/account/ssh-keys.spec.ts"
```

### CODE REVIEW

File: packages/manager/cypress/support/intercepts/profile.ts [hunk 1]
```diff
@@ -421,16 +421,66 @@ export const mockGetSSHKey = (sshKey: SSHKey): Cypress.Chainable<null> => {
...
 /**
- * Intercepts POST request to create an SSH key and mocks the response.
+ * Intercepts POST request to create an SSH key.
  *
- * @param sshKey - SSH key object with which to mock response.
+ * @returns Cypress chainable.
+ */
+export const interceptCreateSSHKey = (): Cypress.Chainable<null> => {
+  return cy.intercept('POST', apiMatcher(`profile/sshkeys*`));
+};
```
Comment (jdamore-linode) [inline packages/manager/cypress/support/intercepts/profile.ts:430]: Are we using this util anywhere?

File: packages/manager/cypress/e2e/core/account/ssh-keys.spec.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  mockDeleteSSHKey,
+  mockGetSSHKeys,
+  mockUpdateSSHKey,
+} from 'support/intercepts/profile';
+import { ui } from 'support/ui';
+import { randomLabel, randomString } from 'support/util/random';
+
+describe('SSH keys', () => {
+  /*
+   * - Vaildates SSH key update flow using mock data.
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/account/ssh-keys.spec.ts:12]:
```suggestion
   * - Validates SSH key update flow using mock data.
```

File: packages/manager/cypress/e2e/core/account/ssh-keys.spec.ts [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+   * - Vaildates SSH key delete flow using mock data.
+   * - Confirms that the dialog opens when clicking.
+   * - Confirms UI flow when user deletes an SSH key.
+   */
+  it('deletes an SSH key via Profile page as expected', () => {
+    const mockSSHKeys = sshKeyFactory.buildList(2);
+
+    mockGetSSHKeys(mockSSHKeys).as('getSSHKeys');
+
+    // Navigate to SSH key landing page, click the "Add an SSH Key" button.
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/account/ssh-keys.spec.ts:116]:
```suggestion
    // Navigate to SSH key landing page.
```

File: packages/manager/cypress/e2e/core/account/ssh-keys.spec.ts [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+    });
+    const newSSHKeyLabel = randomLabel();
+    const modifiedSSHKey = sshKeyFactory.build({
+      ...mockSSHKey,
+      label: newSSHKeyLabel,
+    });
+
+    mockGetSSHKeys([mockSSHKey]).as('getSSHKeys');
+
+    // Navigate to SSH key landing page, click the "Add an SSH Key" button.
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/account/ssh-keys.spec.ts:37]:
```suggestion
    // Navigate to SSH key landing page.
```

**Review by jdamore-linode — APPROVED** [review]
Great work @cliu-akamai, posted a couple really minor suggestions for the comments, but the tests look perfect! Approved pending adding changeset with `yarn changeset` 👍

**Review by AzureLatte — APPROVED** [review]
Test passed

**Pull request merged**